app/views/users/show.html.erb

```diff
@@ -1,11 +1,19 @@
-<div class="row">
+
```

Comment:
will fix as best as I can - I will work on html/css - try and get everything lined up and centered